### PR TITLE
added Russian translation

### PR DESCRIPTION
**Pull request by arzanov**
Add RU locale into index.ts
Fixed some bugs in the ru_RU.ts

### CODE REVIEW

File: packages/core/client/src/locale/index.ts [hunk 1]
```diff
@@ -1,53 +1,67 @@
...
-      client: {
-        ...zhCN,
-      },
-    },
-  },
-} as Record<string, LocaleOptions>;
+import antdEnUS from 'antd/lib/locale/en_US';
+import antdJaJP from 'antd/lib/locale/ja_JP';
+import antdZhCN from 'antd/lib/locale/zh_CN';
+import antdZhCN from 'antd/lib/locale/ru_RU';
```
Comment (chenos) [inline packages/core/client/src/locale/index.ts:4]: ru

File: packages/core/client/src/locale/index.ts [hunk 2]
```diff
@@ -1,53 +1,67 @@
...
+import enUS from './en_US';
+import jaJP from './ja_JP';
+import zhCN from './zh_CN';
+import zhCN from './ru_RU';
```
Comment (chenos) [inline packages/core/client/src/locale/index.ts:8]: ruRU?

Comment (arzanov) [reply packages/core/client/src/locale/index.ts:8]: Oh sorry my ignorance.
Yes, ruRU, thanks.

File: packages/core/client/src/locale/index.ts (hunk 1 above)
Comment (arzanov) [reply packages/core/client/src/locale/index.ts:4]: RuRU

**Review by arzanov — COMMENTED** [review]
import antdRuRU from 'antd/lib/locale/ru_RU';
import ruRU from './ru_RU';

**Pull request merged**